code/obj/machinery/espresso.dm

```diff
@@ -88,6 +88,9 @@
 										playsound(src.loc, 'sound/misc/pourdrink.ogg', 50, 1)
 									return
 						if ("Remove cup")
+							if (get_dist(src, usr) > 1 || isAI(user))
```

Comment:
Seems like this is using the old method of remote access check?

Reply:
I care that we are nearby and physical, so essentially the inversion of old method.